examples/expo-router/nativewind-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="nativewind/types" />
```

Comment:
Add this to remove type errors for className as per the docs: https://www.nativewind.dev/v4/getting-started/typescript